packages/docs/.gitattributes

```diff
@@ -0,0 +1,7 @@
+# Makes Git translate the line ending for the specified extensions while always keeping things with "unix endings" in repo. This means that scripts in this repo won't break if things are checked out and in on a Windows machine.
```

Comment:
discuss: might make more sense to pull this up to the root of the monorepo if we need it.